src/plugins/data/public/ui/query_editor/language_selector.tsx

```diff
@@ -57,8 +56,8 @@ export const QueryLanguageSelector = (props: QueryLanguageSelectorProps) => {
 
   languageService.getLanguages().forEach((language) => {
     if (
-      (language && props.appName && !language.supportedAppNames.includes(props.appName)) ||
-      uiService.Settings.getUserQueryLanguageBlocklist().includes(language?.id)
+      (language && props.appName && !language.editorSupportedAppNames?.includes(props.appName)) ||
```

Comment:
nit: i think we can clean this up. language select should already dispalyed when its in a supported app nap